quantities/factories/ThirdPower.cs

```diff
@@ -8,31 +8,31 @@
 
 namespace Quantities.Factories;
 
-public readonly struct CubicFactory<TQuantity, TCompound, TCubic, TLinear> : ICubicFactory<TQuantity, TCubic, TLinear>
-    where TLinear : Dimensions.IDimension, ILinear
-    where TCubic : ICubic<TLinear>, Dimensions.IDimension
-    where TCompound : ICompoundFactory<TQuantity, TLinear>, IAliasingCreate
+public readonly struct ThirdPower<TCreate, TQuantity, TCubic, TLinear> : ICubicFactory<TQuantity, TCubic, TLinear>
+    where TCreate : struct, ICreate
     where TQuantity : IFactory<TQuantity>
+    where TCubic : ICubic<TLinear>, Dimensions.IDimension
+    where TLinear : Dimensions.IDimension, ILinear
 {
-    private readonly TCompound cubicFactory;
-    public TCompound Cubic => this.cubicFactory;
-    internal CubicFactory(in TCompound compound) => this.cubicFactory = compound;
+    private readonly TCreate create;
+    public Compound<TCreate, TQuantity, TLinear> Cubic => new(in this.create, AllocationFree<PowerInjector<TCreate, Cubic>>.Item);
```

Comment:
Maybe you could name the class ThirdPowerFactory and the injected value "ThirdPower"?

Reply:
See my comment on the equally badly named "SecondPower" (factory).